MAUI/ListView/item-drag-and-drop.md

```diff
@@ -0,0 +1,485 @@
+        </Grid>
+      </Grid>
+    </DataTemplate>
+  </syncfusion:SfListView.ItemTemplate>
+</syncfusion:SfListView>                
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+listView.ItemTemplate = new DataTemplate(() =>
+{
+  var grid = new Grid();
+  var name = new Label
+  {
+    FontSize = 15,
+    VerticalOptions = LayoutOptions.Center
+  };
+  name.SetBinding(Label.TextProperty, new Binding("Name"));
+
+  var indicatorGrid = new Grid()
+  {
+    Padding = new Thickness(10, 20, 20, 20),
+    HorizontalOptions = LayoutOptions.End,
+    VerticalOptions = LayoutOptions.Center
+  };
+  var dragIndicatorView = new DragIndicatorView() { ListView = this.listView };
+  var indicator = new Image() { Source = "dragindicator.png" };
+
+  indicatorGrid.Children.Add(indicator);
+  dragIndicatorView.Content = indicatorGrid;
+
+  grid.Children.Add(name);
+  grid.Children.Add(dragIndicatorView);
+  grid.SetRow(dragIndicatorView, 1);
+  grid.SetColumn(dragIndicatorView, 0);
+  return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+The screenshot shows the output of the reordering items by drag and drop. 
+
+![Item reordering by drag and drop](Images/item-drag-and-drop/maui-listview-dragviewindicator.gif)
+
+## Drag item customization
+
+By defining the `SfListView.DragItemTemplate` property of the `SfListView`, you can display a custom user interface (UI) when performing drag and drop operations. The template can be defined either in code or XAML.
+
+N> If the `BackgroundColor` is set for the `DragItemTemplate` or `DragIndicatorView`, make sure to set `InputTransparent` to `true`. This is necessary because dragging does not occur when using the `DragIndicatorView` in UWP.
+
+{% tabs %}
+{% highlight xaml %}
+<ContentPage xmlns:syncfusion="clr-namespace:Syncfusion.Maui.ListView;assembly=Syncfusion.Maui.ListView">
+  <syncfusion:SfListView x:Name="listView" 
+                   ItemsSource="{Binding ToDoList}"
+                   DragStartMode="OnHold"
+                   BackgroundColor="#FFE8E8EC"
+                   ItemSize="60">
+  <syncfusion:SfListView.DragItemTemplate>
+    <DataTemplate>
+      <Grid Padding="10">
+        <Label x:Name="textLabel" Text="{Binding Name}" FontSize="15" />
+      </Grid>
+    </DataTemplate>
+  </syncfusion:SfListView.DragItemTemplate>
+</syncfusion:SfListView>                
+</ContentPage>
+{% endhighlight %}
+{% highlight c# %}
+listView.DragItemTemplate = new DataTemplate(() => {
+  var grid = new Grid();
+  var name = new Label { FontSize = 15 };
+  name.SetBinding(Label.TextProperty, new Binding("Name"));
+  grid.Children.Add(name);
+  return grid;
+});
+{% endhighlight %}
+{% endtabs %}
+
+## Event
+
+The `ItemDragging` event is raised while dragging and dropping the item in the `SfListView`. The `ItemDraggingEventArgs` has the following members which provides the information for the `ItemDragging` event:
+
+ * `Action`: Returns the drag `Action` such as start, dragging, and drop.
```

Comment:
* `Action`: Return the drag `Action` such as start, dragging, and drop.

Reply:
Addressed the concerns provided.